core/src/main/java/com/digitalpebble/storm/crawler/bolt/ParserBolt.java

```diff
@@ -201,22 +204,24 @@ public void execute(Tuple tuple) {
         }
 
         // parse
+        String text;
         try {
             tika.getParser().parse(bais, teeHandler, md, parseContext);
             text = textHandler.toString();
         } catch (Exception e) {
-            LOG.error("Exception while parsing {}", url, e.getMessage());
+            String errorMessage = "Exception while parsing " + url + ": "
+                    + e.getMessage();
```

Comment:
Internally the compiler will use a StringBuilder anyway so I don't see the value. I will change it to use e.toString() however which includes the class of the exception